snapcraft/cli/_options.py

```diff
@@ -59,6 +59,7 @@ def get_project_options(**kwargs):
             kwargs[key] = value
 
     project_args = dict(
+        debug=ctx.obj['debug'] or kwargs.get('debug'),
```

Comment:
Why is `ctx.obj['debug']` needed here? We have **kwargs** and **ctx.parent.params**. The existing code already handles  the parent context for options that exist on multiple levels.